src/components/AutoSuggest/AutoSuggest.tsx

```diff
@@ -268,6 +268,12 @@ const AutoSuggest: FC<AutoSuggestProps> = ({
     setShowList(hasResults)
   }, [data, numOptionsDeterminer])
 
+  useEffect(() => {
```

Comment:
Ha @kiruys ik dacht er goed aan te doen de lege response melding in een aparte useEffect te doen omdat de andere useEffect hierboven numOptionsDeterminer gebruikt die elders nodig is, maar niet voor het showen van de feedback. Er kan natuurlijk gekozen om nuOptionsDeterminer te refactoren, het doet zo weinig dat het eigenlijk geen eigen functie behoeft.